std/experimental/allocator/mallocator.d

```diff
@@ -64,19 +64,19 @@ struct Mallocator
 }
 
 ///
-unittest
+@nogc unittest
 {
     auto buffer = Mallocator.instance.allocate(1024 * 1024 * 4);
     scope(exit) Mallocator.instance.deallocate(buffer);
     //...
 }
 
-unittest
+@nogc unittest
 {
-    static void test(A)()
+    @nogc static void test(A)()
     {
         int* p = null;
-        p = new int;
+        p = cast(int*) A.instance.allocate(int.sizeof);
```

Comment:
It's a one line fix and there's no reason to leak unless absolutely necessary, so please add it.